pkg/core/combat/icd.go

```diff
@@ -125,6 +127,8 @@ var ICDGroupEleApplicationSequence = [][]int{
 	{1.0, 0.0, 0.0, 0.0},
 	//yelan burst
 	{1.0, 0.0, 0.0, 1.0, 0.0, 0.0, 1.0, 0.0, 0.0, 1.0, 0.0, 0.0, 1.0, 0.0, 0.0, 1.0, 0.0, 0.0, 1.0, 0.0, 0.0, 1.0, 0.0, 0.0},
+	//tighnari
+	{1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0},
```

Comment:
his durability sequence is 1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0

Reply:
also you're missing the damage sequence in the slice below, which should be 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1